File: Data Structures & Algorithms/concatenation-of-array/submission-0.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int n = nums.length;
        int[] ans = new int[n*2];
        System.out.println("ans length = " + ans.length);
        int k = 0;
        for(int i = 0; i < ans.length; i++){

            if (i < n){
                ans[i] = nums[i];
                System.out.println(ans[i]);
            }

            if (i >= n){
                    ans[i] = nums[k];
                    k++;
                    System.out.println(ans[i]);
            }
        }
        return ans;
    }
}

File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        seen = {}
        for num in nums:
            if num in seen:
                seen[num] += 1
            else:
                seen[num] = 1
        
        sorted_items = sorted(seen.items(), key =lambda x : x[1] ,reverse=True)

        return [num for num, freq in sorted_items[:k]]
            